src/features/database-editor/components/code-dataset.jsx

```
@@ -0,0 +1,45 @@
+      >
+        <small>code</small>
+        <div
+          className="cea-database-editor-database-dataset-code-buttons"
+          style={{ display: 'flex', flexDirection: 'column', gap: 12 }}
+        >
+          {Object.keys(data).map((code) => {
+            return (
+              <Button
+                key={code}
+                onClick={() => setCode(code)}
+                type={code === code ? 'primary' : 'default'}
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)